config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
```

Comment:
Again. There are a few files in initializers with just comments. Can they be removed?

Reply:
Again, yep, and actually did start stripping them out. Mostly left them there it make it easier to reinstate the behaviour later. But definitely happy to remove (especially as re-enabling isn't as simple as just uncommenting the code)